cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/DBSink.java

```diff
@@ -176,17 +178,47 @@ private void ensureJDBCDriverIsAvailable() throws Exception {
       LOG.debug("Plugin Type: {} and Plugin Name: {}; Driver Class: {} not found. Registering JDBC driver via shim {} ",
                 dbSinkConfig.jdbcPluginType, dbSinkConfig.jdbcPluginName, driverClass.getName(),
                 JDBCDriverShim.class.getName());
-      driverShim = new JDBCDriverShim(driverClass.newInstance());
+      if (driverShim == null) {
+        driverShim = new JDBCDriverShim(driverClass.newInstance());
```

Comment:
https://github.com/caskdata/cdap/pull/3038#discussion_r33396842 Based on this discussion, I chose to make the driverClass a field so that I could call destroy